CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- 'display' prop to AutocompleteInput
```

Comment:
what do think about the name of the props `size`?

Reply:
just did it! the prop name exposed to `AutocompleteInput` is now `size`. However, I was't able to set the same name for the `SearchInput` component. My guess is it's somehow already used, so I tried to use a more descriptive name (`barSize`, btw)there even though they're not the same